Makefile

```diff
@@ -509,20 +509,20 @@ docker-compose-sources: .git/hooks/pre-commit
 
 
 docker-compose: awx/projects docker-compose-sources
-	docker-compose -f tools/docker-compose/_sources/docker-compose.yml $(COMPOSE_OPTS) up $(COMPOSE_UP_OPTS) --remove-orphans
+	docker compose -f tools/docker-compose/_sources/docker-compose.yml $(COMPOSE_OPTS) up $(COMPOSE_UP_OPTS) --remove-orphans
```

Comment:
On my (relatively) new system, I get:

```
$ docker compose
docker: 'compose' is not a docker command.
```

What version of Docker did this get added? I'm not really in favor of making a change that requires everyone to go update something for not a whole lot of gain.

Reply:
>  I'm not really in favor of making a change that requires everyone to go update something for not a whole lot of gain.

I can totally understand the reason for that. If this is a complete breaking-change, then I'd be cool to write a "if you use compose v2, please set up an alias" note or whatever instead (even though I'm not a big fan of having to build workarounds).

Reply:
maybe have the default be `docker-compose` in a make variable at the top of the makefile, and users can override it with `docker compose` if needed

Reply:
> maybe have the default be `docker-compose` in a make variable at the top of the makefile, and users can override it with `docker compose` if needed

Great idea! Thanks Seth!

I've applied your suggestion and adjusted the commit message too.